pkg/config/config.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"flag"
+	"io/ioutil"
+
+	prom "github.com/grafana/agent/pkg/prometheus"
```

Comment:
Since this new package doesn't import the other Prometheus we don't need the import renaming anymore.